<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchSelectionWindow</class>
 <widget class="QDialog" name="MatchSelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Match</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="archiveCoverContainer" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>350</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>350</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QTableWidget" name="twList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>7</verstretch>
           </sizepolicy>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>4</number>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Series</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Publisher</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Title</string>
           </property>
          </column>
         </widget>
         <widget class="QTextEdit" name="teDescription">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>3</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="altCoverContainer" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>350</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>350</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MatchSelectionWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MatchSelectionWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
